MUXSDKKaltura/MUXSDKKaltura/Models/MUXSDKViewOrientation.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  MUXSDKViewOrientation.swift
+//  Mux-Stats-Kaltura
+//
+//  Created by Stephanie Zuñiga on 4/11/21.
+//  Copyright © 2021 Mux, Inc. All rights reserved.
+//
+
+import Foundation
+
+public enum MUXSDKViewOrientation {
+    case unknown
```

Comment:
wondering if we need the `unknown` case - this method is designed to be called by the app developer and they should know if it is portrait or landscape?

Reply:
I dont think we need it I am removing it.